documentation/docs/faq/liquidity-mining.md

```diff
@@ -100,59 +100,96 @@ The Hummingbot Miners app uses your Ethereum wallet address to:
-    Wallet address are use for receiving payouts only, you do not need deposit assets into or trade using this wallet.
-
-#### Get Ethereum deposit address with Metamask browser extension  
-1. Add Metamask extension to your favorite browser.  
-2. Open Metamask and create an account - complete the sign up process.  
-3. On Metamask dashboard make sure Network is set to Main Ethereum Network (the default network for Ether transaction is Main Net).  
-4. The Ethereum address is located Below Main Ethereum Network along with your name.  
-5. You can copy the Ethereum address by clicking your name.  
+### How to add wallet address?
+
+1. Log in to https://miners.hummingbot.io
+2. Click **Settings**.
+3. Scroll down to **Connect Wallet**, and click down arrow (on the right) to expand the section. The wallet option is displayed.
+4. Choose the wallet you would like to connect base on the campaign you're participating.
+5. For miners participating in the Binance - Zcoin campaign, choose **`Zcoin`** row (background is highlighted in gray when mouse over), enter your Zcoin deposit address.
+6. For miners participating in the KuCoin - Harmony One campaing, choose **`Harmony`** row (background is highlighted in gray when mouse over), enter your Harmony One deposit address.
+7. For miners participating in other campaings on Binance beside Zcoin, choose **`Ethereum`** row (background is highlighted in gray when mouse over), enter your Ethereum deposit address.
+
+!!! note 
+    Wallet address not used for trading. Wallet address are use for receiving payouts only, you do not need to deposit assets into or trade using this wallet.
+
+### Get Ethereum deposit address with Metamask browser extension
+
+1. Add Metamask extension to your favorite browser.
+2. Open Metamask and create an account - complete the sign up process.
+3. On Metamask dashboard make sure Network is set to Main Ethereum Network (the default network for Ether transaction is Main Net).
+4. The Ethereum address is located Below Main Ethereum Network along with your name.
+5. You can copy the Ethereum address by clicking your name.
 
 Above steps are for new user, existing user can jump to step #3 after successful login.
-<br />  
 
-#### Get Ethereum and Zcoin deposit address from Binance  
-1. Go to [Binance](https://www.binance.com/) website.  
-2. Complete the sign up process.  
-3. Navigate to Spot wallet, click the **Deposit** button and choose **XZC** to display the Zcoin deposit address or choose **ETH** to display the Ethereum deposit address. 
+### Get Ethereum and Zcoin deposit address from Binance
 
-Above steps are for new user, existing user can jump to step #3 after successful login.  
+1. Go to [Binance](https://www.binance.com/) website.
+2. Complete the sign up process.
+3. Navigate to Spot wallet, click the **Deposit** button and choose **XZC** to display the Zcoin deposit address or choose **ETH** to display the Ethereum deposit address.
```

Comment:
Take out XZC everywhere since we no longer support Zcoin.

Also generalize this, we don't have to specify ETH.  It's any asset that's a reward asset in a campaign.